release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1148,10 +1154,22 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* Previously, after you imported image streams tags, the `ImageContentSourcePolicy` (ICSP) custom resource (CR) could not co-exist with the `ImageDigestMirrorSet` (IDMS) or `ImageTagMirrorSet` (ITMS) CR. {product-title} would choose ICSP instead of the other CR types. With this release, these custom resources can co-exist, so after you import image stream tags, {product-title} can choose the required CR. (link:https://issues.redhat.com/browse/OCPBUGS-30279[*OCPBUGS-30279*])
```

Comment:
```suggestion
* Previously, after you imported image streams tags, the `ImageContentSourcePolicy` (ICSP) custom resource (CR) could not co-exist with the `ImageDigestMirrorSet` (IDMS) or `ImageTagMirrorSet` (ITMS) CR. {product-title} chose ICSP instead of the other CR types. With this release, these custom resources can co-exist, so after you import image stream tags, {product-title} can choose the required CR. (link:https://issues.redhat.com/browse/OCPBUGS-30279[*OCPBUGS-30279*])
```

Reply:
One past perfect that isn't needed.